modules/Memory.py

```diff
@@ -126,7 +126,8 @@ def __init__(self, pid):
                 os._exit(1)
         time.sleep(0.5)
 
-
+def GetGame():
+    return mGBA.game
```

Comment:
I don't understand the point of this, why don't you just grab this directly from the mGBA class?

Reply:
from where I needed to access it I would need to import some stuff, honestly I wasnt sure if you could import a class or not so I did this so I could just import the required info